.rubocop.yml

```diff
@@ -19,9 +19,16 @@ Metrics/BlockLength:
 Lint/MissingSuper:
   Enabled: false
 
+# Essentially, logs are making methods and blocks huge, not the
+# logic itself. We thus decided to disable some linter checks
+# related to number of lines.
+
 Metrics/MethodLength:
   Enabled: false
 
+Metrics/BlockLength:
```

Comment:
What do you think of disabling in specific snippet instead of the whole project?

Something like that:
```
# rubocop:disable Metrics/MethodLength
def my_code
  ..
end
# rubocop:enable Metrics/MethodLength
```

Reply:
The specific snippet is pretty much anything that contains logs :sweat_smile: Our logs here have lots of lines.

There might be a better way to log, though; it's just that we ran into a lot of linter problems with this early on so we ended up disabling the check

Reply:
Got it! :smiley: